modules/accounting/model/accounting.class.php

```diff
@@ -906,78 +913,83 @@ private function sub_mva_voucher($args)
                 #print_r($fields);
                 $VoucherID2 = $_lib['storage']->db_new_hash($fields, 'voucher');
 
-                #setter link fra fï¿½rste mva postering til mva motpostering
+                #sets link from first vat journal line to vat counterpart journal line
                 $fields2['voucher_AutomaticVatVoucherID'] = $VoucherID2;
-				// use voucher period as mva period
+                // use voucher period as mva period
                 $this->update_voucher_line($fields2, $VoucherID, 'sub_mva_voucher2', $args['voucher']->VoucherPeriod);
             }
 
             ##################################
             elseif($args['action'] == 'update')
             {
-                $_lib['sess']->debug("Update MVA postering");
+                $_lib['sess']->debug("Update VAT journal line");
+                # Automatically updated VAT journal line
                 $fields['voucher_AutomaticReason'] = "Automatisk oppdatert MVA postering $fields[voucher_Vat]";
-                $fields['voucher_DisableAutoVat'] = 1; #setter disable autovat til true, fordi dette er en autovat linje
+                $fields['voucher_DisableAutoVat'] = 1; #sets disable autovat to true, because it is an auto vat line
 
                 $fields['voucher_ProjectID']        = $args['voucher']->ProjectID;
                 $fields['voucher_DepartmentID']     = $args['voucher']->DepartmentID;
                 $fields['voucher_CarID']            = $args['voucher']->CarID;
                 $fields['voucher_VoucherPeriod']     = $args['voucher']->VoucherPeriod;
 
-                #print "Oppdater MVA postering pk VoucherID: " . $args['voucherAV']->VoucherID; print_r($fields) ; print "<br>\n";
+                #print "Updates VAT journal line primary key VoucherID: " . $args['voucherAV']->VoucherID; print_r($fields) ; print "<br>\n";
                 $this->update_voucher_line($fields, $args['voucherAV']->VoucherID, 'sub_mva_voucher3');
 
-                #Motsatt pï¿½ samme konto
+                #Counterpart on same accountplan
                 $fields['voucher_AccountPlanID'] = $args['AccountPlan']->AccountPlanID;
                 $tmpIn                           = $fields['voucher_AmountIn'];
                 $fields['voucher_AmountIn']      = $fields['voucher_AmountOut'];
                 $fields['voucher_AmountOut']     = $tmpIn;
+                # Automatically updated VAT journal line
                 $fields['voucher_AutomaticReason'] = "Automatisk oppdatert MVA motpostering $fields[voucher_Vat]";
 
-                #gjï¿½r ingenting med linken. lar den vï¿½re som den er
+                #does nothin with the link. lets it be like it is
                 //$fields['voucher_AutomaticVatVoucherID'] = $VoucherID;
 
-                if($args['voucherAV']->AutomaticVatVoucherID) #hvis det eksisterer en motpostering oppdateres denne
+                if($args['voucherAV']->AutomaticVatVoucherID) #if the countrpart journal line exists it gets updated
```

Comment:
if the countrpart journal line exists it gets updated
countrpart => counterpart